relayer-engine/src/index.ts

```diff
@@ -79,14 +81,18 @@ export async function run(args: RunArgs): Promise<void> {
       ]);
       break;
     default:
+      logger.error(`Invalid running mode as argument: ${process.env.MODE}`);
```

Comment:
I thought about this but didn't see the value since structured data is something we often want to query on. I can't really imagine a use case where we need to query by the `production`/`testing` tags.

Reply:
hmm, yeah I guess you're right. There could be some use of filtering to just the listener or executor logs, but it seems unlikely to do that rather than more specific queries

Reply:
I could add it as default meta for the root logger, otherwise that meta prop will only appear on this line.

Reply:
I think some of that is already happening in each harness